code/src/pages/cart/components/CartItem.scss

```diff
@@ -0,0 +1,66 @@
+@import "@design-system";
+
+.spa-cart-item {
+  display: flex;
+  align-items: center;
+  padding: spacing(4) 0;
+  border-bottom: 1px solid var(--spa-cart-item-border-color, $gray-light);
+
+  &__image {
+    width: spacing(25);
+    margin-right: spacing(5);
+  }
+
+  &__details {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+  }
+
+  &__title {
+    font-size: spacing(5);
+    margin-bottom: spacing(3);
+  }
+
+  &__quantity-selector {
+    display: flex;
+    align-items: center;
+    border: 1px solid var(--spa-cart-quantity-selector-border-color, $black);
+    padding: spacing(1);
+    margin-right: spacing(5);
+  }
+
+  &__quantity-block {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: spacing(8);
+    height: spacing(8);
+    cursor: pointer;
+  }
+
+  &__quantity-input {
+    width: spacing(10);
+    text-align: center;
+    border: none;
+    outline: none;
+    font-size: spacing(4);
+  }
+
+  &__price {
+    font-weight: var(--spa-cart-price-font-weight, $fw-extra-bold);
+    font-size: spacing(5);

```

Comment:
Use variant and isBold props from appTypography

Reply:
> Use variant and bold variant props from appTypography

Can we set `variant` for `AppBox` ?

Reply:
I didn't catch your question. You are using AppTypography for this case. You can set it to AppTypography
![image](https://github.com/idx-academy/spa-orders/assets/118522791/5f217794-4a29-4a55-98f0-b700e84675df)

Reply:
> I didn't catch your question. You are using AppTypography for this case. You can set it to AppTypography ![image](https://private-user-images.githubusercontent.com/118522791/347165819-5f217794-4a29-4a55-98f0-b700e84675df.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA1NjQ3OTIsIm5iZiI6MTcyMDU2NDQ5MiwicGF0aCI6Ii8xMTg1MjI3OTEvMzQ3MTY1ODE5LTVmMjE3Nzk0LTRhMjktNGE1NS05OGYwLWI3MDBlODQ2NzVkZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcwOVQyMjM0NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0wMDRiZWE1YzI2ZDIzYTZlMzg2ZWNlNTkwNTM4ZjQ5ZjA5YTA4NGM3MjU5YjkyYTY4YzViNDU1M2NkM2QxNjQ0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.VcfThu1tHFsBGBAAGZNwBL_Ou1PtAJY2hgPJFA6evhQ)

But in this case definition like variant not working even with `h1`. Think `component` is enough
<img width="454" alt="image" src="https://github.com/idx-academy/spa-orders/assets/56305508/8b2780fe-4c29-4667-83ff-31b91e737fc8">